litr/src/main/java/com/linkedin/android/litr/resample/UpsampleAudioResampler.java

```diff
@@ -0,0 +1,79 @@
+package com.linkedin.android.litr.resample;
+
+import androidx.annotation.NonNull;
+import java.nio.ByteBuffer;
+
+/**
+ * An {@link AudioResampler} that upsamples from a lower sample rate to a higher sample rate.
+ */
+public class UpsampleAudioResampler extends PassThroughAudioResampler {
+    @Override
+    public void resample(@NonNull ByteBuffer inputBuffer, int inputSampleRate, @NonNull ByteBuffer outputBuffer,
+            int outputSampleRate, int channels) {
+        super.resample(inputBuffer, inputSampleRate, outputBuffer, outputSampleRate, channels);
+        // TODO currently up-sampling not supported because it's need some enhancements, so fallback to PASSTHROUGH approach
```

Comment:
We probably don't want to do this here. I think we can remove this entire class for now and bring it in later.

Reply:
ok